etc/.golangci.yaml

```diff
@@ -0,0 +1,82 @@
+service:
+  golangci-lint-version: 1.51.x
+run:
+  deadline: 900s
+  modules-download-mode: readonly
+linters:
+  enable-all: true
+  disable:
+    - asasalint
+    - containedctx
+    - contextcheck
+    - cyclop
+    - deadcode
+    - exhaustivestruct
+    - exhaustruct
+    - forcetypeassert
+    - funlen
+    - gocognit
+    - godox
+    - goerr113
+    - gochecknoglobals
+    - gochecknoinits
+    - gocyclo
+    - gofmt
+    - goimports
+    - golint
+    - gomnd
+    - ifshort
+    - importas
+    - interfacebloat
+    - interfacer
+    - ireturn
+    - maintidx
+    - maligned
+    - makezero
+    - musttag
+    - nestif
+    - nlreturn
+    - nosnakecase
+    - nonamedreturns
+    - nosprintfhostport
+    - paralleltest
+    - prealloc
+    - scopelint
+    - structcheck
+    - tagliatelle
+    - testpackage
+    - thelper # false positives
+    - varcheck
+    - varnamelen
+    - wrapcheck
+    - wsl
+    - depguard # allow importing of package viamrtsp
+    - gocritic # false positives in cgo calls
```

Comment:
```
decoder.go:103:89: dupSubExpr: suspicious identical LHS and RHS for `==` operator (gocritic)
        ret := C.avformat_open_input(&avFormatCtx, cURL, nil, nil)
                                                                  ^
decoder.go:107:109: dupSubExpr: suspicious identical LHS and RHS for `==` operator (gocritic)
        defer C.avformat_close_input(&avFormatCtx)
                                                  ^
decoder.go:198:80: dupSubExpr: suspicious identical LHS and RHS for `==` operator (gocritic)
                C.av_frame_free(&d.dstFrame)
                                            ^
decoder.go:205:79: dupSubExpr: suspicious identical LHS and RHS for `==` operator (gocritic)
        C.av_frame_free(&d.srcFrame)
                                    ^
decoder.go:217:132: dupSubExpr: suspicious identical LHS and RHS for `==` operator (gocritic)
        res := C.avcodec_send_packet(d.codecCtx, &avPacket)
                                                           ^
decoder.go:231:81: dupSubExpr: suspicious identical LHS and RHS for `==` operator (gocritic)
                        C.av_frame_free(&d.dstFrame)
```

I attributed it to giving false positives since there is not equality operator in any of these lines